File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/issuedTicket/exception/CanNotCancelEntranceException.java
package band.gosrock.domain.domains.issuedTicket.exception;

import band.gosrock.common.exception.DuDoongCodeException;

public class CanNotCancelEntranceException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new CanNotCancelEntranceException();

    private CanNotCancelEntranceException() {
        super(IssuedTicketErrorCode.CAN_NOT_CANCEL_ENTRANCE);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/issuedTicket/exception/CanNotCancelEntranceException.kt
package band.gosrock.domain.domains.issuedTicket.exception

import band.gosrock.common.exception.DuDoongCodeException

class CanNotCancelEntranceException private constructor() : DuDoongCodeException(IssuedTicketErrorCode.CAN_NOT_CANCEL_ENTRANCE) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = CanNotCancelEntranceException()
    }
}
